src/screens/change-wallet/ChangeWalletSheet.tsx

```diff
@@ -200,7 +200,7 @@ export default function ChangeWalletSheet() {
         const visibleAccounts = wallet.addresses.filter(account => account.visible);
         let walletTotalBalance: string | null = null;
 
-        visibleAccounts.forEach(account => {
+        visibleAccounts?.forEach(account => {
```

Comment:
Similarly here, don't see how `visibleAccounts` can be falsy. Wouldn't the line above error first?

```ts
const visibleAccounts = wallet.addresses.filter(account => account.visible);
```